server/command.go

```diff
@@ -341,6 +362,25 @@ func (p *Plugin) webhookCommand(parameters []string, info *gitlab.UserInfo) stri
 		}
 		return fmt.Sprintf("Invalid command: %s", subCommand)
 
+	case commandBuild:
```

Comment:
Why is this command also placed under `webhook`?

Reply:
How else this would be called other wise? if i don't add this here the tests won't run like they does for the other commands

Reply:
Doesn't L325 ff. already contain this code? Why does it need to be duplicated?

Reply:
Line 325 is part of commandAdd switch case

Reply:
Yes, but why does code need to get duplicated?

Reply:
Yes, you are right i can abstract it out in a function, i was just going with the rule of three
https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)